config.json

```diff
@@ -23,6 +23,17 @@
       "unlocked_by": null,
       "uuid": "74515d45-565b-4be2-96c4-77e58efa9257"
     },
+    {
+      "core": false,
+      "difficulty": 6,
+      "slug": "parallel-letter-frequency",
+      "topics": [
+        "concurrency",
+        "type_conversion"
+      ],
+      "unlocked_by": null,
```

Comment:
It would be great it you could choose a core exercise for this to be unlocked by :) Normally we'd choose an easier core exercise with relatively similar topics. If you don't want to choose one that's also fine, we can open an issue for it :)

Reply:
I chose`nucleodtide-count`.  That's what I based my solution on to begin with.  LMK if you think that's too big a leap.

Reply:
It has to be unlocked by a core exercise and `nucleotide-count` isn't a core exercise, sorry. `nucleotide-count` is unlocked by the core exercise `hamming`, so that might be a good one for this to be unlocked by as well?

Reply:
Oh sure. Sorry about the misunderstanding!